.circleci/config.yml

```diff
@@ -3006,69 +3022,38 @@ workflows:
       - pytorch_doc_push:
           requires:
             - pytorch_linux_xenial_cuda8_cudnn7_py3_build
-      - pytorch_linux_xenial_cuda9_cudnn7_py2_build
-      - pytorch_linux_xenial_cuda9_cudnn7_py2_test:
-          requires:
-            - pytorch_linux_xenial_cuda9_cudnn7_py2_build
-      - pytorch_linux_xenial_cuda9_cudnn7_py3_build
-      - pytorch_linux_xenial_cuda9_cudnn7_py3_test:
-          requires:
-            - pytorch_linux_xenial_cuda9_cudnn7_py3_build
-      - pytorch_linux_xenial_cuda9_2_cudnn7_py3_gcc7_build
-      - pytorch_linux_xenial_cuda9_2_cudnn7_py3_gcc7_test:
-          requires:
-            - pytorch_linux_xenial_cuda9_2_cudnn7_py3_gcc7_build
-      - pytorch_linux_xenial_cuda10_cudnn7_py3_gcc7_build
 
       # Pytorch MacOS builds
       - pytorch_macos_10_13_py3_build
       - pytorch_macos_10_13_py3_test:
           requires:
             - pytorch_macos_10_13_py3_build
-      - pytorch_macos_10_13_cuda9_2_cudnn7_py3_build
 
-      # Caffe2 builds
+      # Caffe2 Linux builds
       - caffe2_cmake_cuda9_0_cudnn7_ubuntu16_04_build
       - caffe2_cmake_cuda9_0_cudnn7_ubuntu16_04_test:
           requires:
             - caffe2_cmake_cuda9_0_cudnn7_ubuntu16_04_build
-      - caffe2_py2_cuda9_0_cudnn7_ubuntu16_04_build
```

Comment:
This is actually the only TensorRT build https://github.com/pytorch/pytorch/blob/master/.jenkins/caffe2/build.sh#L137

Reply:
A little related to this, we should eliminate all string checks on the entire config name like this, and include all needed config info the job name. cc @kostmo